packages/cli/src/oclif/commands/migrate.js

```diff
@@ -46,7 +47,9 @@ class MigrateCommand extends BaseCommand {
     }
 
     const body = {
-      percent,
+      name: 'migrate',
+      from_version: fromVersion,
+      to_version: toVersion,
```

Comment:
Is this the right place to review map from? 
https://github.com/zapier/zapier/blob/16c0ad7ce5dbb39f9802bd8b45c2305c35b77699/web/backend/zapier/app_platform/developer_cli/api/migration_resources.py

Reply:
yep!